apps/librepcb/main.cpp

```diff
@@ -82,6 +82,24 @@ int main(int argc, char* argv[])
     // Initialize all 3rd party libraries
     init3rdPartyLibs();
 
+    // Migrate from old setting format to INI
+    // backward compatibility - remove this some time!
```

Comment:
Maybe mark it with a `TODO`. Most editors and IDE's will highlight those.

Reply:
Too late... I already added exactly the same comment at several other locations (unfortunately without TODO), now I use the same comment just for consistency. I will remove them anyway after the first release...